src/wallet/walletdb.h

```diff
@@ -95,13 +105,34 @@ class CKeyMetadata
 
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
+        if (ser_action.ForRead()) {
+            SetNull();
+        }
         READWRITE(this->nVersion);
         nVersion = this->nVersion;
         READWRITE(nCreateTime);
         if (this->nVersion >= VERSION_WITH_HDDATA)
         {
+            // Core/Knots 0.13+
```

Comment:
Stale comment, will get rid of it in the rebase. (I decided against including this in [Knots](http://bitcoinknots.org/) 0.13 to avoid further incompatibilities.)